### ISSUE TITLE
New nightly build

### ISSUE THREAD

**Issue by HOSTED-POWER** [mRemoteNG/mRemoteNG#2228]
Hi,

As a normal user, it's not easy to compile mRemoteNG ourselves, could you please release a new nightly build? Perhaps even schedule this to happen once a month or similar? :)

**Comment by Kvarkas** [comment]
I will think that is possible to do about such, but you should know nb is not kind of beta release its may have some changes or preparation for bigger changes so even if functionality stay in place but no warranty what it will work as expected (it may drop weird error or show strange messages etc)

**Comment by HOSTED-POWER** [comment]
I agree, but it's better than 0 updates for really long periods. Current nb has bugs too, we can always revert to another nightly build if there are several available :)

**Comment by Kvarkas** [comment]
ok, will try to prepare kind of auto deploy via appveyer like portable version so should be easy to download and test, but even if I try to don't make any danger stuff in it, you will use on you own risk ;)

**Comment by HOSTED-POWER** [comment]
Wow that sounds fantastic, thanks a lot!!

We run on the nigthly from here: https://mremoteng.org/download

And we use it with MySQL, so it would be great if the updated compile can be simply replaced. Not sure which version was there.

**Comment by Kvarkas** [comment]
its states version on download page ;) yes that suppose too be updated automatically on that page

**Comment by simonai1254** [comment]
For me nightly build may have bugs (maybe there ~is~ will be a section in the Change Notes about Known Issues) and the preview build should contain a more ore less stable version to test new features and sort out bugs before they are included in the stable build... 

With this I guess nothing should prevent you to create automated builds. 

One might just want to use the feature branches even a bit more to prevent broken builds, but I guess this will be sorted out more or less soon :)

**Comment by HOSTED-POWER** [comment]
Perhaps any update on a nightly build? Or even better some new final release :)

**Comment by tecxx** [comment]
~~i would be happy if the settings-save-bug would be fixed. any ETA on that?~~

fixed it myself. see PR

**Comment by eebssk1** [comment]
There's no stable prebuilt version of 1.77.2.
You have to build it yourself.

**Comment by tecxx** [comment]
deleted

**Comment by LIyoai** [comment]
#2245

**Comment by robertpopa22** [comment]
**Fix available for testing**

A fix for this issue has been implemented in our community fork (`robertpopa22/mRemoteNG`, branch `main`). A beta build including this fix is available from the fork's Releases page.

Please test and report if this resolves your issue.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
